dotnet/src/webdriver/Internal/ResourceUtilities.cs

```diff
@@ -114,6 +116,40 @@ public static Stream GetResourceStream(string fileName, string resourceId)
 
     private static string GetPlatformString()
```

Comment:
Here I just took historical implementation of platform determination. `SeleniumManager.cs` also has his own determination. It would be nice to unify this logic and use one good entry point to determine platform.